### PR TITLE
[stable/insights-agent] update kyverno job rbac

### PR DESCRIPTION
**Pull request by Azahorscak**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Review by rbren — APPROVED** [review]
Thanks Adam! CC @jslivka

File: stable/insights-agent/templates/kyverno/rbac.yaml [hunk 1]
```diff
@@ -14,20 +14,29 @@ metadata:
     app: insights-agent
 rules:
 - apiGroups:
-  - '*'
+  - kyverno.io
```
Comment (jslivka) [inline stable/insights-agent/templates/kyverno/rbac.yaml:17]: I'm indifferent if we want to wildcard the whole group or not, but we will also need to include the `wgpolicyk8s.io` api group

```suggestion
  - kyverno.io
  - wgpolicyk8s.io
```

Comment (jslivka) [reply stable/insights-agent/templates/kyverno/rbac.yaml:17]: That's for these CRDs:

```
clusterpolicyreports.wgpolicyk8s.io
policyreports.wgpolicyk8s.io
```

**Review by jslivka — APPROVED** [review]

**Comment by jslivka** [comment]
LGTM

**Pull request merged**